pass HIP_CLANG_INCLUDE_PATH which is determined by clang -print-resource-dir
===================================================================
--- HIP-rocm-5.1.3.orig/bin/hipcc.pl
+++ HIP-rocm-5.1.3/bin/hipcc.pl
@@ -105,6 +105,7 @@ $ROCM_PATH      =   $hipvars::ROCM_PATH;
 $HIP_VERSION    =   $hipvars::HIP_VERSION;
 $HSA_PATH       =   $hipvars::HSA_PATH;
 $HIP_ROCCLR_HOME =   $hipvars::HIP_ROCCLR_HOME;
+$HIP_CLANG_INCLUDE_PATH =   $hipvars::HIP_CLANG_INCLUDE_PATH;
 
 if ($HIP_PLATFORM eq "amd") {
   # If using ROCclr runtime, need to find HIP_ROCCLR_HOME
